src/app/project/project.component.ts

```diff
@@ -229,6 +229,17 @@ export class ProjectComponent implements OnInit {
         this._router.navigate(['/'], { relativeTo: this._route });
     }
 
+    /**
+     * given an Html element, compare the scrollHeight and the clientHeight
+     * determines whether or not the tooltip should be displayed
+     *
+     * @param elem the element which has the line-clamp css
+     * @returns inverse of comparison between the scrollHeight and the clientHeight of elem
+     */
+    hideTooltip(elem: HTMLElement): boolean {
+        return !(elem.scrollHeight > elem.clientHeight);
+    }
```

Comment:
see above about renaming function

Reply:
done in [1b17e0b](https://github.com/dasch-swiss/dsp-app/pull/790/commits/1b17e0b73acd424ad941d9f83fa89f51660dcd5e)